iam-proxy-italia-project/wwwallet/configs/.env.prod

```diff
@@ -0,0 +1,28 @@
+HOST='0.0.0.0'
+PORT=3000
+VITE_WS_URL=ws://localhost/wwwallet-server/
+VITE_WALLET_BACKEND_URL=https://localhost/wwwallet-server/
```

Comment:
please replace `localhost` using the ENV var about the hostname configured in the project